src/wp-includes/class-wp-plugin-dependencies.php

```diff
@@ -369,10 +369,15 @@ public static function get_dependency_data( $slug ) {
 	 */
 	public static function display_admin_notice_for_unmet_dependencies() {
 		if ( in_array( false, self::get_dependency_filepaths(), true ) ) {
+			$plugins_url = is_multisite() ? network_admin_url( 'plugins.php' ) : admin_url( 'plugins.php' );
 			wp_admin_notice(
-				__( 'There are additional plugin dependencies that must be installed.' ),
+				sprintf(
+					/* translators: %s: Link to the plugins page. */
+					__( 'Some required plugins are missing or inactive. Please <a href="%s">review your plugins</a>.' ),
```

Comment:
Will this notice display anywhere other than at the top of the Plugins page? If not, I don't think a link is necessary. The only exception, I think, would be if you're on a site within a network. Then I think it makes sense to link to the network admin plugin page, but only if the user has that capability.

Reply:
It'll appear on `Plugins > Installed Plugins` and `Plugins > Add New`